google-cloud-security_center/synth.py

```diff
@@ -18,280 +18,22 @@
-    '"README.md", "LICENSE"',
-    '"README.md", "AUTHENTICATION.md", "LICENSE"'
-)
-s.replace(
-    '.yardopts',
-    'README.md\n',
-    'README.md\nAUTHENTICATION.md\nLICENSE\n'
-)
-
-# https://github.com/googleapis/google-cloud-ruby/issues/3058
-s.replace(
-    'google-cloud-security_center.gemspec',
-    '\nGem::Specification.new do',
-    'require File.expand_path("../lib/google/cloud/security_center/version", __FILE__)\n\nGem::Specification.new do'
-)
-s.replace(
-    'google-cloud-security_center.gemspec',
-    '(gem.version\s+=\s+).\d+.\d+.\d.*$',
-    '\\1Google::Cloud::SecurityCenter::VERSION'
-)
-s.replace(
-    'lib/google/cloud/security_center/v*/*_client.rb',
-    '(require \".*credentials\"\n)\n',
-    '\\1require "google/cloud/security_center/version"\n\n'
-)
-s.replace(
-    'lib/google/cloud/security_center/v*/*_client.rb',
-    'Gem.loaded_specs\[.*\]\.version\.version',
-    'Google::Cloud::SecurityCenter::VERSION'
-)
-
-# Fix links for devsite migration
-for file in ['lib/**/*.rb', '*.md']:
-    s.replace(
-        file,
-        'https://googleapis.github.io/google-cloud-ruby/#/docs/google-cloud-logging/latest/google/cloud/logging/logger',
-        'https://googleapis.dev/ruby/google-cloud-logging/latest'
-    )
-s.replace(
-    '*.md',
-    'https://googleapis.github.io/google-cloud-ruby/#/docs/.*/authentication',
-    'https://googleapis.dev/ruby/google-cloud-security_center/latest/file.AUTHENTICATION.html'
-)
-s.replace(
-    'lib/**/*.rb',
-    'https://googleapis.github.io/google-cloud-ruby/#/docs/.*/authentication',
-    'https://googleapis.dev/ruby/google-cloud-security_center/latest/file.AUTHENTICATION.html'
-)
-s.replace(
-    'README.md',
-    'github.io/google-cloud-ruby/#/docs/google-cloud-security_center/latest/.*$',
-    'dev/ruby/google-cloud-security_center/latest'
-)
-
-# Generate the helper methods
-subprocess.call('bundle update && bundle exec rake generate_partials', shell=True)
-
-# Deal with weirdness where RunAssetDiscoveryResponse is defined in a separate
-# file that doesn't get required properly.
-s.replace(
-    'lib/google/cloud/security_center/v1/security_center_client.rb',
-    '\nrequire "google/cloud/security_center/v1/securitycenter_service_pb"\n',
-    '\nrequire "google/cloud/security_center/v1/securitycenter_service_pb"\nrequire "google/cloud/security_center/v1/run_asset_discovery_response_pb"\n'
-)
-s.replace(
-    'lib/google/cloud/security_center/v1p1beta1/security_center_client.rb',
-    '\nrequire "google/cloud/security_center/v1p1beta1/securitycenter_service_pb"\n',
-    '\nrequire "google/cloud/security_center/v1p1beta1/securitycenter_service_pb"\nrequire "google/cloud/security_center/v1/run_asset_discovery_response_pb"\n'
-)
+gapic = gcp.GAPICMicrogenerator()
+library = gapic.ruby_library(
+    "securitycenter", "v1",
+    generator_args={
+        "ruby-cloud-gem-name": "google-cloud-security_center",
+        "ruby-cloud-title": "Cloud Security Command Center",
```

Comment:
I don't see "Cloud Security Command Center" used anywhere in the official docs, just "Security Command Center". https://cloud.google.com/security-command-center

Reply:
Good catch. Yes, it should just be "Security Command Center". Fixed and regenerated.